src/commands/install.ts

```diff
@@ -26,40 +26,51 @@ export default class Install extends Command {
 
   static args = [];
 
-  async run() {
-    try {
-      const { flags } = this.parse(Install);
-      let process_path = process.cwd();
-      if (flags.prefix) {
-        process_path = path.isAbsolute(flags.prefix) ?
-          flags.prefix :
-          path.join(process_path, flags.prefix);
+  static async getTasks(root_service_path: string, recursive?: boolean): Promise<Listr.ListrTask[]> {
```

Comment:
This method should be in a common file instead of an individual command since it's called by multiple commands.

Reply:
Good call. Was going to define on our base Command and model after the Command.run func.